client/scss/components/_body_content.scss

```diff
@@ -45,3 +45,54 @@
     }
   }
 }
+
+$large-width: (5/7 * 100)+'%';
+$narrow-width: (5/7) * 100%; // calculates the narrow column width as a percentage of the main column.
+$center-width-large: (6/7) * 100%; // calculates the halfway point of the container, as a percentage of the main column.
+$center-width-xlarge: (6/6) * 100%;
+
+.body-part {
+  display: block;
+}
+
+// TODO: Update to be like figure--full-width, and then generify
+.body-part--full-width {
+  @include respond-to('medium') {
+    margin-left: -18%;
+    margin-right: -18%;
+  }
+
+  @include respond-to('large') {
+    clear: both;
+    position: relative;
+    z-index: 1;
+    margin-left: 0;
+    margin-right: calc((#{$large-width} + #{map-get($gutter-width, 'large')})* -1);
+    background-color: color('cream');
+    padding-top: $vertical-space-unit;
+    margin-top: -$vertical-space-unit;
+    border-bottom: $vertical-space-unit solid color('cream');
+    /* stylelint-disable declaration-no-important */
+    opacity: 1 !important;
+    /* stylelint-enable */
+  }
+}
+
+.body-part__scroller {
+  overflow: auto;
+
+  @include respond-to('large') {
+    margin-left: -60px;
```

Comment:
It actually comes from the container layout, I've updated it:
https://github.com/wellcometrust/wellcomecollection.org/pull/285/commits/5e4290e7c18384f67247e94aabb596f42cfd3e41